.github/workflows/docs-workflow.yml

```diff
@@ -32,61 +32,23 @@ jobs:
         with:
           ruby-version: '3.4.2'
           bundler-cache: true
-          working-directory: design-system-docs
+          working-directory: website
 
       # Install dependencies
       - uses: bahmutov/npm-install@v1
         with:
-          working-directory: design-system-docs
+          working-directory: website
 
-      - run: ./bin/bridgetown deploy
-        working-directory: design-system-docs
+      - run: ./bin/static-build
+        working-directory: website
 
       - name: Deploy draft to Netlify
         uses: South-Paw/action-netlify-deploy@v1.2.1
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
           netlify-auth-token: ${{ secrets.NETLIFY_AUTH_TOKEN }}
           netlify-site-id: ${{ secrets.NETLIFY_SITE_ID }}
-          build-dir: ./design-system-docs/output
+          build-dir: ./website/build
           draft: true
           comment-on-pull-request: true
           config-path: ./netlify.toml
-
-  # This job will:
-  #   * deploy a production build every time there is a release created
-  #   * comment that commit with the deploy URL
-  publishOnRelease:
```

Comment:
I'm unsure why this has been removed, was this dead code (as in, docs-workflow-main achieves the same end?)

Reply:
Oops, mean to drop a comment on it that this is redundant as it gets overridden every time we merge to main.